src/cow-react/modules/limitOrders/containers/LimitOrdersWidget/index.tsx

```diff
@@ -148,10 +156,12 @@ export function LimitOrdersWidget() {
                 showSetMax={showSetMax}
                 topLabel={inputCurrencyInfo.label}
               />
-              <styledEl.RateWrapper>
-                <RateInput />
-                <DeadlineInput />
-              </styledEl.RateWrapper>
+              {!isWrapOrUnwrap && (
```

Comment:
cool, i like this solution a lot. Feels natural to hide this info when I tried it.